apps/src/makerlab/dropletConfig.js

```diff
@@ -18,30 +18,35 @@ const playSongConfig = {
-  {func: 'analogRead', parent: api, category: 'Maker Lab', type: 'value', nativeIsAsync: true, params: ['5']},
+  {func: 'pinMode', parent: api, category: 'Maker Lab', paletteParams: ['pin', 'mode'], params: ['13', '"output"'], dropdown: { 1: ['"output"', '"input"', '"analog"'] }},
+  {func: 'digitalWrite', parent: api, category: 'Maker Lab', paletteParams: ['pin', 'value'], params: ['13', '1'], dropdown: { 1: ['1', '0'] }},
+  {func: 'digitalRead', parent: api, category: 'Maker Lab', type: 'value', nativeIsAsync: true, paletteParams: ['pin'], params: ['"D4"']},
+  {func: 'analogWrite', parent: api, category: 'Maker Lab', paletteParams: ['pin', 'value'], params: ['5', '150']},
+  {func: 'analogRead', parent: api, category: 'Maker Lab', type: 'value', nativeIsAsync: true, paletteParams: ['pin'], params: ['5']},
 
-  {func: 'on', blockPrefix: 'led.', category: 'Circuit', tipPrefix: pixelType, modeOptionName: "*.on", objectDropdown: { options: simpleLedVariables, dropdownOnly: true } },
-  {func: 'off', blockPrefix: 'led.', category: 'Circuit', tipPrefix: pixelType, modeOptionName: "*.off", objectDropdown: { options: simpleLedVariables, dropdownOnly: true }  },
-  {func: 'toggle', blockPrefix: 'led.', category: 'Circuit', tipPrefix: pixelType, modeOptionName: "*.toggle", objectDropdown: { options: simpleLedVariables, dropdownOnly: true }  },
-  {func: 'blink', blockPrefix: 'led.', category: 'Circuit', paletteParams: ['period'], params: ['50'], tipPrefix: pixelType, modeOptionName: "*.blink", objectDropdown: { options: simpleLedVariables, dropdownOnly: true }  },
-  {func: 'stop', blockPrefix: 'led.', category: 'Circuit', tipPrefix: pixelType, modeOptionName: "*.stop", objectDropdown: { options: simpleLedVariables, dropdownOnly: true }  },
-  {func: 'intensity', blockPrefix: 'pixel0.', category: 'Circuit', params: ['25'], tipPrefix: pixelType, modeOptionName: "*.intensity", objectDropdown: { options: colorPixelVariables, dropdownOnly: true }  },
-  {func: 'color', blockPrefix: 'pixel0.', category: 'Circuit', paletteParams: ['color'], params: ['"#FF00FF"'], tipPrefix: pixelType, modeOptionName: "*.color", objectDropdown: { options: colorPixelVariables, dropdownOnly: true }  },
+  {func: 'led', category: 'Circuit', type: 'readonlyproperty', noAutocomplete: true},
+  {func: 'led.on', category: 'Circuit'},
+  {func: 'led.off', category: 'Circuit'},
 
-  {func: 'piezo', category: 'Circuit', type: 'readonlyproperty', noAutocomplete: true},
-  {func: 'piezo.frequency', category: 'Circuit', params: ['500', '100']},
-  {func: 'piezo.note', category: 'Circuit', params: ['"A4"', '100']},
-  {func: 'piezo.stop', category: 'Circuit'},
-  {func: 'piezo.play', category: 'Circuit', paletteParams: ['song'], params: [JSON.stringify(playSongConfig)]},
+  {func: 'on', blockPrefix: colorLedBlockPrefix, category: 'Circuit', tipPrefix: pixelType, modeOptionName: '*.on', objectDropdown: {options: colorPixelVariables}},
```

Comment:
(Note: these array accessor blocks aren't drop-down-able yet, so the `objectDropdown` property is just a placeholder.)